src/platforms/esp32/components/libatomvm/CMakeLists.txt

```diff
@@ -21,7 +21,8 @@
 idf_component_register(INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}/../../../../libAtomVM")
 
 add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../../../../libAtomVM" "libAtomVM")
-target_include_directories(${COMPONENT_LIB} INTERFACE "${CMAKE_CURRENT_BINARY_DIR}/libAtomVM")
```

Comment:
Hm, how are we picking up version.h, then if we remove this?

Reply:
Using transitivity on dependency on libAtomVM target I believe. Otherwise, it wouldn't compile, would it?

libAtomVM/CMakeLists.txt, line 158.